src/api/firebase/deleteFirestore.ts

```diff
@@ -0,0 +1,15 @@
+import { firestore } from './firebase';
+import { doc, deleteDoc, deleteField, updateDoc } from 'firebase/firestore';
+
+export const deleteUserFirestore = async (uid: string) => {
+  await deleteDoc(doc(firestore, 'users', uid));
+};
+
+export const deleteFieldsFirestore = async (collection: string, docId: string, field: string) => {
```

Comment:
Nu inteleg care este scopul acestei functii. ce sterge/updateaza?

Reply:
Sterge un anume field din obiectul stocat in firestore nu intregul obiect. Cealalata functie sterge tot obiectul cu id-ul din parametrii.

Reply:
ok am inteles